source/patterns/@aws-solutions-constructs/aws-kinesisstreams-gluejob/README.md

```diff
@@ -0,0 +1,151 @@
+| fieldSchema?        | [`CfnTable.ColumnProperty[]`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-glue.CfnTable.ColumnProperty.html) | Glue Database for this construct. If not provided the construct will create a new Glue Database. The database is where the schema for the data in Kinesis Data Streams is stored                                                                                                      |
+| database?           | [`CfnDatabase`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-glue.CfnDatabase.html)                           | Glue Database for this construct. If not provided the construct will create a new Glue Database. The database is where the schema for the data in Kinesis Data Streams is stored                                                                                                      |
+| table?              | [`CfnTable`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-glue.CfnTable.html)                                 | Glue Table for this construct, If not provided the construct will create a new Table in the database. This table should define the schema for the records in the Kinesis Data Streams. Either @table or @fieldSchema is mandatory. If @table is provided then @fieldSchema is ignored |
+| outputDataStore?    | [`SinkDataStoreProps`](#sinkdatastoreprops)                                                                                   | The output datastore properties where the ETL output should be. Current datastore types suported is only S3. written                                                                                                                                                                  |
+
+## SinkDataStoreProps
+
+| **Name**        | **Type**                                                                                | **Description**                                                                                                                                                                                                               |
+| :-------------- | :-------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| s3OutputBucket? | [`Bucket`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-s3.Bucket.html) | The output S3 location where the data should be written. The provided S3 bucket will be used to pass the output location to the etl script as an argument to the AWS Glue job. If not provided, the construct will create one |
```

Comment:
name should start with existing

If we accept an existing bucket, we should also accept a BucketProps object.

Reply:
updating interface prop names